.jenkins/dsl/jobs.groovy

```diff
@@ -217,8 +217,14 @@ void setupOptaPlannerTurtleTestsJob(String jobFolder) {
     def jobParams = getJobParams('optaplanner-turtle-tests', jobFolder, 'Jenkinsfile.turtle',
             'Run OptaPlanner turtle tests on a weekly basis.')
     KogitoJobTemplate.createPipelineJob(this, jobParams).with {
-        triggers {
```

Comment:
I trust it's ok. Is there a new DSL reference I could check? All I found seems to refer to the previous syntax:
https://www.jenkins.io/doc/book/pipeline/syntax/#triggers

Reply:
Thanks for the link, I was not aware of this "api-viewer".